src/test/java/org/jenkinsci/plugins/workflow/graph/FlowNodeTest.java

```diff
@@ -486,6 +486,15 @@ public void nodeWithNoParentsInBruteForceScanForEnclosingBlock() {
         });
     }
 
+    @Test
```

Comment:
Just to check, does this test fail without the fix? Also, can you please add `@Issue` so we know that this is a regression test?

```suggestion
    @Issue("JENKINS-64438")
    @Test
```

Reply:
I can it locally and I can confirm that without the fix the test fails

Reply:
Yes - despite the commit order I did run the test without the fix to check it failed :)